administration/src/bp-modules/cards/ImportCardsController.tsx

```diff
@@ -0,0 +1,101 @@
+import { NonIdealState, Spinner } from '@blueprintjs/core'
+import { ReactElement, useCallback, useContext } from 'react'
+import { useNavigate } from 'react-router-dom'
+
+import { WhoAmIContext } from '../../WhoAmIProvider'
+import CSVCard from '../../cards/CSVCard'
+import { Region } from '../../generated/graphql'
+import { ProjectConfigContext } from '../../project-configs/ProjectConfigContext'
+import useBlockNavigation from '../../util/useBlockNavigation'
+import StandaloneCenter from '../StandaloneCenter'
+import GenerationFinished from './CardsCreatedMessage'
+import CreateCardsButtonBar from './CreateCardsButtonBar'
+import ImportCardsInput, { FILE_SIZE_LIMIT_MEGA_BYTES } from './ImportCardsInput'
+import CardImportTable from './ImportCardsTable'
+import useCardGenerator, { CardActivationState } from './hooks/useCardGenerator'
+
+const FileRequirementsText = `Die Datei darf maximal ${FILE_SIZE_LIMIT_MEGA_BYTES} MB groß sein und muss im CSV Format sein.`
```

Comment:
The limit is currently 2MB, which the table can easily handle, but I think the PDF generation would struggle. We can add an additional limit in number of entries in thi PDF. Maybe 50 entries are still reasonable? (results in around 8MB pdf)

Reply:
i think 50 entries are absolutely fine!

Reply:
It seems we agreed to 100 lines for CSV files with Nürnberg, so I'll go with 100 lines